cypress.config.js

```diff
@@ -5,10 +5,14 @@ const createEsbuildPlugin = require('@badeball/cypress-cucumber-preprocessor/esb
 const createBundler = require('@bahmutov/cypress-esbuild-preprocessor')
 const { chromeAllowXSiteCookies } = require('@dhis2/cypress-plugins')
 const { defineConfig } = require('cypress')
+// const {
```

Comment:
We should just remove this commented code. At which point we need this, we can easily add it.